Test/Integration/CarrierTest.php

```diff
@@ -0,0 +1,72 @@
+<?php
+
+declare(strict_types=1);
+
+namespace StoreKeeper\StoreKeeper\Test\Integration;
+
+use Magento\Framework\DataObject;
+use Magento\Framework\ObjectManagerInterface;
+use Magento\Framework\TestFramework\Unit\Helper\ObjectManager;
+use Magento\Quote\Model\Quote\Address\RateRequest;
+use Magento\Quote\Model\Quote\Address\RateResult\Error;
+use Magento\Quote\Model\Quote\Address\RateResult\Method;
+use Magento\Shipping\Model\Rate\Result;
+use Magento\TestFramework\Helper\Bootstrap;
+use StoreKeeper\StoreKeeper\Api\OrderApiClient;
+use StoreKeeper\StoreKeeper\Model\Carrier\Storekeeper as StorekeeperCarrier;
+
+class CarrierTest extends CarrierTestAbstract
```

Comment:
I do not understand why there is `CarrierTestAbstract` , the child just call parent in all cased. Looks like unneded abstraction.

Reply:
I do not really agree it's more readable in our  case, but I'll accept as it follows magento style.